### ISSUE TITLE
Unable to specify additional request params to slides.presentations.get with typescript

### ISSUE THREAD

**Issue by wrakky** [googleapis/google-api-nodejs-client#1363]
I need to fetch all the URLs of images in a presentation using the slides API and want to use the `fields` parameter in my request so that only the image urls are returned and not the entire presentation data.

Using plain javascript the following works fine:
```js
const slides = google.slides({
  version: 'v1',
  auth: oauth2Client
});
const presentation = await slides.presentations.get({
  presentationId,
  fields: 'slides(pageElements(image(contentUrl)))'
});
```

However, doing the same this with Typescript does not compile because the params argument for this call is defined by `Params$Resource$Presentations$Get` which only includes an `auth` and `presentationId` key so I'm unable to add `fields` here.

If i try to use the additional Axios options parameter (as suggested in the docs) this compiles but the axios params are ignored and `fields` is not included in the request:
```js
const presentation = await slides.presentations.get(
  { presentationId },
  { 
    params: {
      fields: 'slides(pageElements(image(contentUrl)))'
    }
  }
});
```

I can get around the issue by casting the original params to `any` but there seems to either be an issue with either the Typescript definitions or the Axios option params merging depending on what the desired way to use this API is.

#### Environment details
  - OS: Ubuntu
  - Node.js version: v10.5
  - npm version: 6.4.1
  - `googleapis` version: 33.0.0

**Comment by neckaros** [comment]
Same problem with Drive API i have to cast to any everytime. 
Ex: Params$Resource$Files$List
Maybe those object should extend a basic type with common properties?

**Comment by ajaaym** [comment]
This is fixed in this pr [1390](https://github.com/googleapis/google-api-nodejs-client/pull/1390)

**Comment by ajaaym** [comment]
Fix is released in this version [v35.0.0](https://github.com/googleapis/google-api-nodejs-client/releases/tag/v35.0.0)
